src/defaults.js

```diff
@@ -9,7 +9,7 @@
  * OF ANY KIND, either express or implied. See the License for the specific language
  * governing permissions and limitations under the License.
  */
-const DEFAULT_PATTERNS = ['src/**/*.htl'];
+const DEFAULT_PATTERNS = ['src/**/*.htl', 'src/**/*.js'];
```

Comment:
I assume there is no risk of confusion between `*.pre.js` and `*.js` because only the former get picked up by `ProxyJSAsset.js`.

Reply:
yes and no. it works differently now :-)

- the script generated by the `HelixAsset` asset type will contain a `require` to the `pre.js` would normally include (bundle) it in the parent script. the `ProxyJSAsset` will mark it as _external_ to prevent that
- the `.pre.js` is treated like a normal script (the problem with this and pure-js scripts will be tackled later in #334)
- finally in the `RawJSPackager`, the scripts are written _raw_ (i.e. without the parcel pre/post fix that contain the bunder loader logic) to the output directory.